meta-mender-core/classes/mender-install-ubi.bbclass

```diff
@@ -0,0 +1,25 @@
+ARTIFACTIMG_FSTYPE ?= "ubifs"
+
+MENDER_STORAGE_DEVICE ?= "ubi0"
```

Comment:
Actually there is a work-around for above, https://lists.freedesktop.org/archives/systemd-devel/2017-January/038197.html.

But do not know if we need it.